docs/architecture/diagrams/get.gov registrar deployment.puml

```diff
@@ -0,0 +1,65 @@
+@startuml get.gov registrar deployment
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Deployment.puml
+
+LAYOUT_WITH_LEGEND()
+title get.gov registrar deployment
+skinparam linetype polyline
+
+Person(team, "get.gov developer", "Code Writer")
+
+Deployment_Node(aws, "AWS GovCloud", "Amazon Web Services Region") {
+    Deployment_Node(cloudgov, "cloud.gov", "Cloud Foundry PaaS") {
+        System_Ext(cloudgov_router, "cloud.gov router", "Cloud Foundry service")
+        System_Ext(cloudgov_uaa, "cloud.gov authentication", "Cloud Foundry service")
+        System_Ext(cloudgov_controller, "cloud.gov controller", "Cloud Foundry orchestration")
+        System_Ext(cloudgov_dashboard, "cloud.gov dashboard", "Cloud Foundry web UI")
+        System_Ext(cloudgov_logdrain, "logs.fr.cloud.gov", "ELK")
+        Boundary(atob, "ATO boundary") {
+            Deployment_Node(organization, "get.gov organization") {
+                Deployment_Node(unstable, "unstable space") {
+                    System_Boundary(dashboard_unstable, "get.gov registrar") {
+                        Container(getgov_app_unstable, "Registrar Application", "Python, Django", "Delivers static HTML/CSS and forms")
```

Comment:
This is good here, but the png is outdated, maybe just needs `git add`?

Reply:
updated!